W-21522534-Salesforce-Pub/Sub-Connector-OAuth-2.0-feedback-sis (#3196)
mulesoft/docs-connectors: salesforce-pubsub/1.2/modules/ROOT/pages/salesforce-pubsub-connector-reference.adoc

Subscribes to a streaming channel. This source provides channel-related notification settings for new events that occur after you subscribe.

[NOTE]
====

> . A channel must be published to Salesforce before a subscription to the channel is created.

Reviewer: ```suggestion
* A channel must be published to Salesforce before a subscription to the channel is created.
```

> . When using OAuth 2.0 authentication, the listener is not initialized during the initial application startup. You must complete the OAuth authorization process and then restart the application to successfully initialize the listener and begin receiving events.

Reviewer: ```suggestion
* When using OAuth 2.0 authentication, the source isn't initialized during the initial application startup. You must complete the OAuth authorization process and then restart the application to successfully initialize the source and begin receiving events.
```
